### PR TITLE
Create mantis-detect.yaml

### PR DESCRIPTION
**Pull request by MaKyOtOx**
### Template / PR Information

Mantis Bug Tracker portal detection
- References: https://www.mantisbt.org/

### Template Validation

I've validated this template locally?
- [X] YES
- [ ] NO

### Additional References:
- [Nuclei Template Creation Guideline](https://nuclei.projectdiscovery.io/templating-guide/)
- [Nuclei Template Matcher Guideline](https://github.com/projectdiscovery/nuclei-templates/wiki/Unique-Template-Matchers)
- [Nuclei Template Contribution Guideline](https://github.com/projectdiscovery/nuclei-templates/blob/master/.github/CONTRIBUTING.md)
- [PD-Community Discord server](https://discord.gg/projectdiscovery)

### CODE REVIEW

**Comment by geeknik** [comment]
Hello, thank you for the submission. A check for MantisBT currently exists in the `tech-detect.yaml` template here: 

https://github.com/projectdiscovery/nuclei-templates/blob/3b400caab08676cedbefe117ecbed8f8f302e61d/technologies/tech-detect.yaml#L1843

File: exposed-panels/mantis-detect.yaml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+  name: Mantis portal detection
+  author: makyotox
+  severity: info
+  tags: panel
+
+requests:
+  - method: GET
+    path:
+      - "{{BaseURL}}/login_page.php"
+    matchers:
```
Comment (toufik-airane) [inline exposed-panels/mantis-detect.yaml:13]:
```suggestion
      - "{{BaseURL}}/login_page.php"

    matchers:
```

**Review by toufik-airane — CHANGES_REQUESTED** [review]
Dear @MaKyOtOx, thank you for your contribution.

@geeknik I won't be shocked to see another Tech detection in the _exposed-panels_ folder, what do you think?

**Comment by geeknik** [comment]
> @geeknik I won't be shocked to see another Tech detection in the _exposed-panels_ folder, what do you think?

I'm all for new templates! 👍🏻

**Comment by MaKyOtOx** [comment]
I wrote this template because the existing once did not detect my Mantis target. It's up to you to add a new template or refine the existing one

**Comment by ehsandeep** [comment]
Thank you @MaKyOtOx for sharing the additional template.

**Comment by toufik-airane** [comment]
Dear @MaKyOtOx,

Your template has been merged.

Thank you for your contribution and we hope to read you soon. :)

**Comment by MaKyOtOx** [comment]
Thank Team !! More PR are coming :)

**Pull request merged**